site/blog/2022/nodejs-deploy-codepipeline.mdx

```diff
@@ -0,0 +1,226 @@
+---
+slug: deploy-nodejs-app-with-codepipeline
+title: Deploy a NodeJS App on AWS using CodePipeline
```

Comment:
thinking more about this people might not be familiar with CodePipeline, how about?

```suggestion
title: Deploy a NodeJS App on AWS using EC2
```